test/e2e/config/vsphere-ci.yaml

```diff
@@ -8,11 +8,11 @@
 # For creating local images, run ./hack/e2e.sh
 
 images:
-  - name: gcr.io/k8s-staging-cluster-api/cluster-api-controller-amd64:v0.3.9
+  - name: gcr.io/k8s-staging-cluster-api/cluster-api-controller-amd64:v0.3.11
```

Comment:
Image preload works only if you pre-pull/build v0.3.11 images on the host machine, otherwise, you can remove this entry in the image list (it will pul for internet in both cases).
Same for all the other v0.3.11 images

Reply:
@fabriziopandini - I think it makes sense to try and load locally first, to avoid pulling if not necessary (it's useful for CI infra connectivity debugging)

Reply:
Fine for me. 
But given that AFAIK you are not pre-pull/building v0.3.11 images locally this operation fails, and so I assume it will always pull during init